projects/sunbird-pdf-player/src/lib/pdf-viewer/pdf-viewer.component.spec.ts

```diff
@@ -1,16 +1,16 @@
-import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { ComponentFixture, TestBed, async } from '@angular/core/testing';
 
 import { PdfViewerComponent } from './pdf-viewer.component';
 
 describe('PdfViewerComponent', () => {
   let component: PdfViewerComponent;
   let fixture: ComponentFixture<PdfViewerComponent>;
 
-  beforeEach(waitForAsync(() => {
```

Comment:
As part of migration to latest version it change to waitForAsync, what is the reason for reverting this change ?

Reply:
Yes, waitForAsync is correct change , due to my local changes it was breaking for me , now i have updated with same change.